src/new/wallet.js

```diff
@@ -230,6 +230,89 @@ class HathorWallet extends EventEmitter {
     }
   }
 
+  /**
+   * Get information of a given address
+   *
+   * @typedef {Object} AddressInfoOptions
+   * @property {string} token Optionally filter transactions by this token uid (Default: HTR)
+   *
+   * @typedef {Object} AddressInfo
+   * @property {number} total_amount_received Sum of the amounts received
+   * @property {number} total_amount_sent Sum of the amounts sent
+   * @property {number} total_amount_available Amount available to transfer
+   * @property {number} total_amount_locked Amount locked and thus no available to transfer
+   * @property {number} token Token used to calculate the amounts received, sent, available and locked
+   * @property {number} index Derivation path for the given address
+   *
+   * @param {string} address Address to get information of 
+   * @param {AddressInfoOptions} options Optional parameters to filter the results
+   *
+   * @return {AddressInfo} Aggregated information about the given address
+   *
+   */
+  getAddressInfo(address, options = {}) {
+    storage.setStore(this.store);
+    const { token = HATHOR_TOKEN_CONFIG.uid } = options;
+
+    // Throws an error if the address does not belong to this wallet
+    if (!this.isAddressMine(address)) {
+      throw new Error('Address does not belong to this wallet.');
+    }
+
+    // Derivation path index
+    const index = wallet.getAddressIndex(address);
```

Comment:
You should use `this.getAddressIndex(address)`

Reply:
Done